package.json

```diff
@@ -32,7 +32,8 @@
     "docs:build:dev": "cd docs && yarn && yarn build:dev",
     "docs:watch": "cd docs && yarn && yarn dev",
     "docs:deploy": "cd docs && yarn && yarn deploy",
-    "test": "jest --coverage && yarn run lint",
+    "test": "jest --coverage && yarn run lint && yarn run test:size",
```

Comment:
Probably aside from this PR but I think that the `test` script owns too many responsibilities.

Reply:
Fairly nitpicky, but this is on purpose. Running `yarn test` will verify everything, if you just want to run the tests, I always just run `yarn jest`. We can align on that over all libraries though